File: sprint1/Java/lesson7code3.java
public static List<Integer> getSmallerPrimes(int n) {
    List<Integer> smallerPrimes = new ArrayList<>();
    for (int num = 2; num <= n; num++) {
        if (isPrime(num)) {
            smallerPrimes.add(num);
        }
    }
    return smallerPrimes;
}

File: sprint1/C++/lesson7code3.cpp
#include <iostream>
#include <vector>
using namespace std;

vector<int> get_smaller_primes(int n) {
    vector<int> smaller_primes;
    for (int num = 2; num <= n; num++) {
        if (is_prime(num)) {
            smaller_primes.push_back(num);
        }
    }
    return smaller_primes;
}
